build/steps-build.yml

```diff
@@ -147,6 +170,27 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: "${{ parameters.pathToSrc }}/coverage.cobertura.xml"
 
+- task: CopyFiles@2
+  displayName: 'Copy AAB'
+  inputs:
+    Contents: '**/*.aab'
```

Comment:
I think you want to filter with the `-Signed`

Reply:
Do you have an example of that?

Reply:
Not here, in appcenter task my bad :p